corehq/apps/registration/templates/registration/partials/start_trial_modal.html

```diff
@@ -6,30 +6,59 @@
     <div class="modal-content">
       <div class="modal-header">
         <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">{% trans 'Close' %}</span></button>
-        <h4 class="modal-title">
-          {% trans "Get a CommCare Professional trial" %}
-        </h4>
+        <h2 class="text-center">
+          <strong id="start-trial-modal-header">
+          {% trans "Request a CommCare Professional trial" %}
+          </strong>
+        </h2>
       </div>
       <div class="modal-body form-horizontal">
 
         {# Hubspot Form #}
         <div class="form form-horizontal"
-             id="get-trial-cta-form-content"></div>
+             id="get-trial-cta-form-content" ></div>
+        
+        {# Schedule Call Choices #}
+        <div id="choose-callback-options" hidden>
+          <p class="margin-top-l text-center lead">
+          {% blocktrans %}
+            Starting your trial requires a brief call with our team to set up your CommCare project space. Please choose if you prefer us to call you back ASAP, or if you would like to book a  time for the call?
+          {% endblocktrans %}
+          </p>
+          <div class="text-center margin-top-xl margin-bottom-xl">
```

Comment:
why are we introducing totally brand new styling for this? can we please follow the existing style?